tests/scenarios/compat-addon-styles-test.ts

```diff
@@ -132,3 +132,83 @@ appScenarios
       });
     });
   });
+
+function setAddonNameAndCSS(addon: Project, letter: string) {
+  addon.pkg.name = `addon-${letter}`;
+  merge(addon.files, {
+    addon: {
+      styles: {
+        'addon.css': `.addon-${letter} {}`,
+      },
+    },
+  });
+}
+
+appScenarios
+  .map('compat-addon-styles', project => {
```

Comment:
seems to work like this but this name should probably be different from the above tests?